pages/common/plocate.md

```diff
@@ -1,6 +1,7 @@
 # plocate
 
 > Find filenames quickly.
+> Make sure to use the `sudo updatedb` to include new files.
```

Comment:
```suggestion
> Make sure to run `sudo updatedb` to include new files.
```

Reply:
It's not getting committed as I've already removed 'the' from that line.

Reply:
You might have to suggest the changes to the latest version of the file